sphinxcontrib_trio/__init__.py

```diff
@@ -80,12 +72,7 @@
 try:
     from async_generator import isasyncgenfunction
```

Comment:
if we're Python 3.6+ do we need `async_generator` still?

Reply:
Not sure. Maybe we do, in order to detect uses of `async_generator`? There's no reason to do that in Python 3.6+ but I guess many projects still use that instead of async for.

@njsmith Can you please take a look?

Reply:
Merging anyway as we can always address this later. @sethmlarson Thanks for the review!